nsxt/utils.go

```diff
@@ -662,3 +662,25 @@ func handlePagination(lister func(*paginationInfo) error) (int64, error) {
 
 	return total, nil
 }
+
+func getContextSchema(forceNew bool) *schema.Schema {
```

Comment:
When do we expect forceNew to be false?
Since context triggers object path change, seems to me ForceNew should always be defined

Reply:
Indeed. Thanks :)

Reply:
I'm guessing that the whole context - no matter what we'll add to it in the future, should be ForceNew as well.